Parser/asdl_c.py

```diff
@@ -1604,14 +1604,14 @@ def visitModule(self, mod):
             value_repr = PyObject_Repr(value);
         }
 
-        Py_DECREF(value);
-
         if (!value_repr) {
             Py_DECREF(name);
             Py_DECREF(value);
             goto error;
         }
 
+        Py_DECREF(value);
```

Comment:
This fix isn't wrong, but I'm going to be nitpicky :smile: 

There's no need to move the `DECREF` to here, we could just get rid of the extra `DECREF` in the error case.

Reply:
Indeed you're right, nice catch!